ChangeLog.md

```diff
@@ -1,5 +1,181 @@
 # Changelog for haskell-language-server
 
+## 1.7.0.0
+
+- Distribute dynamically linked binaries for HLS to avoid statically linking against GLIBC
+  and system libraries, and to avoid unpredictable failures due to subtle differences
+  between the GHC used to compile HLS and the GHC installed on the users machine
+  (@hasufell, #2675, #2431)
```

Comment:
We may need to change https://github.com/haskell/haskell-language-server/blob/9fca35d7277ff4e459b39e0c503931c9339fe394/docs/troubleshooting.md#static-binaries this doc as well

Reply:
I don't think that is needed, since the same caveats about static binaries apply. The difference is that this time we aren't distributing static binaries but are distributing dynamic binaries. If someone tries to compile static binaries on their own then the caveat still applies (and they will get a warning popup).

Reply:
@wz1000 I mean, the document mentions "local building" as the easiest way to get a dynamic binary, but now it's probably easier to just download the binary.